### PR TITLE
Expressjs

### PR DESCRIPTION
**Pull request by ChrisF999**
Added the backend and a very simple example.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,89 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./KoalaBot/windows_Koala.db"
```
Comment (3eak) [inline prisma/schema.prisma:6]: so if we inserted the production database here, it should all just figure itself out?

File: express/routes/verfied_email.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const router = Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.get("/email", async (req, res) => {
+  const verified_email = await prisma.verified_emails.findUnique({
+    where: { u_id: 1 },
+  });
+  res.json(verified_email);
+});
```
Comment (3eak) [inline express/routes/verfied_email.js:11]: can you add one more example of a post? to show how someone would add a new entry to a database?

File: prisma/schema.prisma [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  email_suffix String
+}
+model to_re_verify {
+  u_id Int @id
+  r_id Int
+}
+model verified_emails{
+  u_id Int @id
+  email String
+}
```
Comment (3eak) [inline prisma/schema.prisma:89]: new line at end of file please :)

File: express/routes/verfied_email.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const { Router } = require("express");
```
Comment (3eak) [inline express/routes/verfied_email.js:1]: typo in the file name: `verified_email.js`

Could we just change this file to be called `verify.js` instead? As we would likely have a route for each cog and their backend calls.

File: express/express.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const port = 3001;
+const email = require("./routes/verfied_email.js");
+app.get("/", async (req, res) => {
+  res.send(
+    `Hello Koala! <br> Docs can be found at <a href="https://expressjs.com/en/4x/api.html">Express</a> and <a href="https://www.prisma.io/docs/">Prisma</a>`
+  );
+});
```
Comment (3eak) [inline express/express.js:9]: Do we need this?

**Review by 3eak — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (ChrisF999) [reply prisma/schema.prisma:6]: As long as our schemas stays the same yeah that is just using one made the koala submodule but the link to any koala bot database should work

File: prisma/schema.prisma (hunk 3 above)
Comment (ChrisF999) [reply prisma/schema.prisma:89]: Fairly sure my formatter removed it let me change that

File: express/express.js (hunk 5 above)
Comment (ChrisF999) [reply express/express.js:9]: nope

File: express/routes/verfied_email.js (hunk 4 above)
Comment (ChrisF999) [reply express/routes/verfied_email.js:1]: Yeah the file can be called whatever and opps for the typo just fixed that

File: express/routes/verfied_email.js (hunk 2 above)
Comment (ChrisF999) [reply express/routes/verfied_email.js:11]: sure I can do that now

File: express/express.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+//the port could be loaded from a .env right now it is set to 3001 one higher then the react frontend.
+
+//helmet is a middleware package that sets a range oh http headers with secutry as a focus.
+const helmet = require("helmet")
+app.use(helmet())
+
+//loads the routes
+const verfy = require("./routes/verfy.js");
+
+app.use("/", verfy);
```
Comment (3eak) [inline express/express.js:14]: typos on verify

**Review by 3eak — CHANGES_REQUESTED** [review]
some minor typos

**Comment by ChrisF999** [comment]
Ready for review

File: src/components/Dashboard.js [hunk 7]
```diff
@@ -3,7 +3,16 @@ import "../styling/home.css";
 import { Timeline } from "react-twitter-widgets";
 import { ThemeContext } from "../context/theme";
 
+//Example of making a request to the backend.
+import axios from "axios";
 const Dashboard = () => {
+  React.useEffect(() => {
```
Comment (3eak) [inline src/components/Dashboard.js:9]: Can you import the `useEffect` hook at the top (similar to `useContext`).

Could you also comment out this function? (As it is just an example)

**Review by 3eak — CHANGES_REQUESTED** [review]
One final request

**Review by 3eak — APPROVED** [review]

**Pull request merged**